### PR TITLE
feat: Added "!insult" command

### PR DESCRIPTION
**Pull request by sabinprj**
This command will pick a random insult, and send on the channel. You can
also select the person to insult too.
Example:
```
!insult
!insult @someone
!insult @foo @bar
```

### CODE REVIEW

File: data/insults.txt [hunk 1]
```diff
@@ -0,0 +1,61 @@
+Just what do you think you're doing Dave?
```
Comment (ncatelli) [inline data/insults.txt]: These insults are terrible.

Comment (sabinprj) [reply data/insults.txt]: We can always edit it though.

**Comment by sabinprj** [comment]
Due to lack of better insults, I am cancelling the PR. Will submit again after I find a good list of insults.

**Comment by bramz** [comment]
So as we discussed on discord, feature functionality could be something similar to the following.

`!insult` output random insult at the user who requested insult
`!insult @user` output random insult at specified user
`!insult add <insult here>` add insult to database
`!insult del <id>` delete insult from database

Add a table to migrations, and use `postgres` or you can simply append to the flat file, your choice.

**Comment by bramz** [comment]
Consider this schema, extend upon this or simply add it to migrations.
```
CREATE TABLE insults (
  id serial primary key,
  creator text not null,
  insult text not null
);
```

File: apophis/cmds/insult.py [hunk 2]
```diff
@@ -9,6 +9,7 @@
 !insult add a new insult
 !insult del <integer>
 ```"""
+limit = 500  # Not more than 500 characters insult
```
Comment (bramz) [inline apophis/cmds/insult.py]: You should declare this as a constant.

Comment (sabinprj) [reply apophis/cmds/insult.py]: Okay

**Pull request merged**